.travis.yml

```diff
@@ -1,9 +1,23 @@
 language: php
 
-php: [5.3, 5.4, 5.5, 5.6, hhvm, 7]
+php:
+  - 5.4
+  - 5.5
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - nightly
+
+matrix:
+  include:
+    - php: 5.3
+      dist: precise
+  allow_failures:
+    - php: nightly
 
 before_script:
-  - composer install --dev --prefer-source
+  - composer install -n --prefer-source
```

Comment:
Is the `-n` (aka `--no-interaction`) option really needed? The Symfony Console component behind CLI user interaction usually is clever enough to figure out, that user can't respond to any of questions.

Reply:
Quite the opposite in fact. Since in automated travis build there is no human user to respond to any questions the `-n` (or `--no-interaction`) ensures that it stays that way.

What I'm saying is that such option isn't needed at all, because it's applied automatically.

Why have you added it? Have you experienced any issues with `composer install` command attempting to ask something and that failed the build?

Reply:
In some [cases](https://stackoverflow.com/questions/49812880/how-can-i-run-composer-without-user-interaction), it will happen that issue.

Reply:
Does any of these happen on Travis CI where you've using that option?

Reply:
In my experience, I have not got this happen.
But nobody can guarantee this will not happen in the future.

Reply:
Then please remove unnecessary stuff, like that `-n` option.

Every commit needs to solve an issue. Since we're not having any issue with that (composer asking something during install), then no need to solve it.